### ISSUE TITLE
proc_open() expects parameter 1 to be string, array given

### ISSUE THREAD

**Issue by romovs** [CouscousPHP/Couscous#227]
Happens with PHP 7.2.11
Worked previously with PHP 7.1.23
```
$ couscous preview
Generating /home/rom/projects/docs to /home/rom/projects/docs/.couscous/generated
PHP Warning:  proc_open() expects parameter 1 to be string, array given in phar:///usr/local/bin/couscous/vendor/symfony/process/Process.php on line 285
          
  [Symfony\Component\Process\Exception\RuntimeException]  
  Unable to launch a new process.                         
                                                          
preview [--target TARGET] [--livereload [LIVERELOAD]] [--config CONFIG] [--] [<address>] [<source>]
```

**Comment by kanaikimi** [comment]
I'm getting this too, on PHP 7.1.24 and PHP 7.2.12.

**Comment by rkling01** [comment]
I have recently received the same error - PHP 7.1.16

**Comment by dantleech** [comment]
It seems this can be fixed by updating to Symfony Console ^3.0 which passes a string to `proc_open`, the locked version seems to pass an `array`. Not sure why this has become an issue.
